src/android/ForeSeeAPI.java

```diff
@@ -915,29 +918,50 @@ public boolean invoke(final JSONArray args, final CallbackContext callback, Cord
                     mDigitalCallbacks.clear();
 
                     //2. 
-                    Digital.setDigitalListener(new FSCordovaDigitalListener());
+                    Digital.setDigitalListener(new XMCordovaDigitalListener());
```

Comment:
This should probably be as follows to fit in with other SDKs
```suggestion
                    Digital.setDigitalListener(new EXPCordovaDigitalListener());
```